### PR TITLE
some chat history test.

### PR DESCRIPTION
**Pull request by erezvit**
This is an initial test to try out the lang chain api for chat history.

### CODE REVIEW

**Review by erezvit1 — COMMENTED** [review]
Is this your first PR?

**Comment by erezvit** [comment]
yes, in thi project.

File: main.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+{chat_history}
+Human: {human_input}
+Chatbot:"""
+
+prompt = PromptTemplate(
+    input_variables=["chat_history", "human_input"], template=template
+)
+memory = ConversationBufferMemory(memory_key="chat_history")
+
+llm_chain = LLMChain(
```
Comment (erezvit1) [inline main.py:15]: Can we do it differently?

File: main.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    llm=OpenAI(),
```
Comment (erezvit1) [inline main.py:16]: What is this line doing?

File: main.py (hunk 1 above)
Comment (erezvit) [reply main.py:15]: I couldn't find a better option.

File: main.py (hunk 2 above)
Comment (erezvit) [reply main.py:16]: This line create a new chain.

**Review by erezvit1 — APPROVED** [review]

**Comment by erezvit** [comment]
First lang chain test.

**Pull request merged**